go.mod

```diff
@@ -0,0 +1,3 @@
+module github.com/roc-project/roc-go
+
+go 1.14
```

Comment:
We actually support at least go 1.12 and it's even checked in our travis build.

Reply:
alright